cli/types/cypress.d.ts

```diff
@@ -2995,6 +2995,12 @@ declare namespace Cypress {
     setupNodeEvents: (on: PluginEvents, config: PluginConfigOptions) => Promise<PluginConfigOptions | void> | PluginConfigOptions | void
 
     indexHtmlFile: string
+
+    /**
+     * Enables the "Run All Specs" UI feature, allowing the execution of multiple specs at once (only available for e2e).
```

Comment:
Just curious, why isn't this available for component testing?

Reply:
Requirements were to just implement the `Run All Specs` as is. I don't think it would be too complicated to get it to work with component but it won't work out of the box. There would need to be some wiring to communicate the selected specs to the dev-server process to make sure the specs are included in the compilation. A bit easier with e2e since the compilation is done on demand

Reply:
Gottcya, wasn't sure what the expectations / standards were around supporting new features/experiments in both e2e & ct now that CT has gone GA.

Reply:
@lmiller1990 maybe...but studio was also added before CT went GA. It'd be worth bringing the converstation up in our cross-team sync.

Reply:
Definitely a good thing to bring up in the sync! There's an issue already for adding the CT piece: https://github.com/cypress-io/cypress/issues/24585

> wasn't sure what the expectations / standards were around supporting new features/experiments in both e2e & ct now that CT has gone GA.

We've got some notes on "release stages" from a doc we hoped to finalize around CT release, @rockindahizzy is working on a docs PR that will put those into a form we can discuss in detail. I don't think we have anything in there about minimum expectation for what counts as an "experiment" - seems to be very case-by-case. But we could add this kind of thing there.

In the spirit of getting the feature out so that people who miss "Run All Specs" can get it back, it seemed reasonable to split the CT part up so people can use it sooner. Being "e2e only" is just one of the many problems the 9.x implementation has.

I suspect if we were designing Run All from scratch today, we would not release this implementation even as an experiment, we'd solve more of the problems first. So even if we set particular standards, "Run All Specs" probably is an exception - restoring something we know is broken but that many people found useful even in its broken state.